src/cryptonote_core/tx_pool.cpp

```diff
@@ -59,27 +59,31 @@ namespace cryptonote
   namespace
   {
     /*! The Dandelion++ has formula for calculating the average embargo timeout:
-                          (-k*(k-1)*hop)/(2*log(1-ep))
+                          (-k*(k-1)*hop)/(2*ln(1-ep))
         where k is the number of hops before this node and ep is the probability
         that one of the k hops hits their embargo timer, and hop is the average
         time taken between hops. So decreasing ep will make it more probable
         that "this" node is the first to expire the embargo timer. Increasing k
         will increase the number of nodes that will be "hidden" as a prior
         recipient of the tx.
 
-        As example, k=5 and ep=0.1 means "this" embargo timer has a 90%
-        probability of being the first to expire amongst 5 nodes that saw the
+        NOTE: The paper says `2*log(1-ep)`, however if you read the explanation
+        in b.5 it is clear they meant `ln`.
+
+        As example, k=10 and ep=0.1 means "this" embargo timer has a 90%
+        probability of being the first to expire amongst 10 nodes that saw the
         tx before "this" one. These values are independent to the fluff
         probability, but setting a low k with a low p (fluff probability) is
         not ideal since a blackhole is more likely to reveal earlier nodes in
         the chain.
 
-        This value was calculated with k=5, ep=0.10, and hop = 175 ms. A
+        This value was calculated with k=10, ep=0.10, and hop = 175 ms. A
         testrun from a recent Intel laptop took ~80ms to
         receive+parse+proces+send transaction. At least 50ms will be added to
         the latency if crossing an ocean. So 175ms is the fudge factor for
-        a single hop with 39s being the embargo timer. */
-    constexpr const std::chrono::seconds dandelionpp_embargo_average{CRYPTONOTE_DANDELIONPP_EMBARGO_AVERAGE};
+        a single hop with 1/75 being the embargo _rate_. The average time
+        to blackhole fluff will be 75/10 or 7.5 seconds. */
```

Comment:
The average time depends on where in the stem the black hole happened, 7.5s assumes the black hole happens after the 10th node

Reply:
Fixed.